maro/simulator/scenarios/vm_scheduling/business_engine.py

```diff
@@ -111,11 +115,19 @@ def _load_configs(self):
 
         self._delay_duration: int = self._config.DELAY_DURATION
         self._buffer_time_budget: int = self._config.BUFFER_TIME_BUDGET
+        self._ticks_per_hour: float = self._config.TICKS_PER_HOUR
         # Oversubscription rate.
         self._max_cpu_oversubscription_rate: float = self._config.MAX_CPU_OVERSUBSCRIPTION_RATE
         self._max_memory_oversubscription_rate: float = self._config.MAX_MEM_OVERSUBSCRIPTION_RATE
         self._max_utilization_rate: float = self._config.MAX_UTILIZATION_RATE
 
+        # Price parameters
+        self._price_per_cpu_cores_per_hour: float = self._config.PRICE_PER_CPU_CORES_PER_HOUR
+        self._price_per_memory_per_hour: float = self._config.PRICE_PER_MEMORY_PER_HOUR
+        self._unit_energy_price_per_kwh: float = self._config.UNIT_ENERGY_PRICE_PER_KWH
+        self._power_usage_efficiency: float = self._config.POWER_USAGE_EFFICIENCY
+        self._w_per_kw: int = self._config.W_PER_KW
```

Comment:
You can update the unit of energy consumption to KWH if needed. Or mark the unit as W and use directly use 1000 in the expression.

Such a constant is unnecessary.